src/js/app.js

```
@@ -1,1405 +1,41 @@
-      const aliases = (LANGUAGE_MAP[label] || [label]).map(normalizeTag);
-      return aliases.some(alias => orgTags.has(alias));
-    });
-  } else {
-    // OR logic: org must have ANY selected language
-    return [...selectedLanguages].some(label => {
-      const aliases = (LANGUAGE_MAP[label] || [label]).map(normalizeTag);
-      return aliases.some(alias => orgTags.has(alias));
-    });
-  }
-}
+  for(const o of ORGS){
```

ESLint

[error] 'ORGS' is not defined.

(no-undef)